src/System.Data.DataSetExtensions/src/System/Data/DataRowExtensions.cs

```diff
@@ -176,13 +172,13 @@ private static T ValueField(object value)
                 return (T)value;
             }
 
-            private static TElem? NullableField<TElem>(object value) where TElem : struct
+            private static T NullableField(object value)
```

Comment:
This is now identical to `ReferenceField` and you can collapse them and simplify `Create()`